app/features-json/projects_json_controller.rb

```diff
@@ -1,9 +1,10 @@
 require_relative "../shared/authenticated_controller_base"
 require_relative "./view_models/project_summary_view_model"
+require_relative "./view_models/project_view_model"
 
 module FastlaneCI
   # Controller for providing all data relating to projects
-  class ProjectJSONController < AuthenticatedControllerBase
+  class ProjectsJSONController < AuthenticatedControllerBase
```

Comment:
Ah nice catch. This means we should also change all the other controllers, like `BuildController`, right?

Reply:
@KrauseFx in this case, I think the controller manages multiple `project` objects, whereas the `BuildController` only deals with 1 at a time.

Reply:
Ahh, looks like I'm wrong, it's already renamed